pkg/apis/componentconfig/types.go

```diff
@@ -863,3 +863,31 @@ type AdmissionPluginConfiguration struct {
 	// +optional
 	Configuration runtime.Object
 }
+
+// ResourceQuotaConfiguration provides configuration for the ResourceQuota
+// admission controller.
+type ResourceQuotaConfiguration struct {
+	metav1.TypeMeta
+
+	// LimitedResources whose consumption is limited by default.
+	// +optional
+	LimitedResources []LimitedResource
+}
+
+// LimitedResource matches a resource whose consumption is limited by default.
+// To consume the resource, there must exist an associated quota that limits
+// its consumption.
+type LimitedResource struct {
+	// API group of the object.
+	// +optional
+	APIGroup string
+
+	// API resource of the object.
+	APIResource string
+
+	// If a resource name tracked by quota for this entity matches this expression, and there is
```

Comment:
didn't see this coming.  Do we have a use-case for different quota enforcement based on what the user names something? That seems unpredictable and error prone.